plugin/src/main/kotlin/de/triplet/gradle/play/internal/Listings.kt

```diff
@@ -28,15 +28,20 @@ internal enum class ListingDetail(val fileName: String, val maxLength: Int = Int
     DEFAULT_LANGUAGE("defaultLanguage");
 }
 
-internal enum class ImageType(val fileName: String, val maxNum: Int = 8) {
-    ICON("icon", 1),
-    FEATURE_GRAPHIC("featureGraphic", 1),
-    PROMO_GRAPHIC("promoGraphic", 1),
+// Min length for any side: 320px. Max length for any side: 3840px.
+internal data class ImageSize(val minWidth: Int, val minHeight: Int, val maxWidth: Int = minWidth, val maxHeight: Int = minHeight)
+
+internal val screenshotSize = ImageSize(320, 320, 3840, 3840)
+
+internal enum class ImageType(val fileName: String, val constraints: ImageSize = screenshotSize, val maxNum: Int = 8) {
```

Comment:
Not sure if we want to create a new data class Constraints, that can hold any possible constraints:
- Min amount / Max amount
- Min dimensions / Max dimensions
- Alpha allowed (e.g. icon) or forbidden (e.g. screenshots)
- Anything more?